chunks/lang-go/chunk.yaml

```diff
@@ -8,3 +8,6 @@ variants:
   - name: "1.18.2"
     args:
       GO_VERSION: 1.18.2
```

Comment:
```suggestion
```
Why are you keeping `lang-go:1.18.2` here? That looks like debris. `1.18.0` and `1.18.1`, too.